pkg/gui/presentation/commits_test.go

```diff
@@ -72,6 +74,54 @@ func TestGetCommitListDisplayStrings(t *testing.T) {
 		sha2 commit2
 						`),
 		},
+		{
+			testName: "show local branch head",
```

Comment:
I'd be more explicit about what's being tested here. Am I right in saying we're testing to ensure we _don't_ get the marker on a main branch or on the head commit?

Reply:
Yes; I changed the title in 1032a05a7601.